jbdevstudio/multiple/jbdevstudio-multiple.target

```diff
@@ -44,11 +39,6 @@
       <unit id="org.apache.commons.io" version="2.0.1.v201105210651"/>
       <unit id="org.apache.commons.codec" version="1.3.0.v201101211617"/>
       <unit id="org.apache.commons.httpclient" version="3.1.0.v201012070820"/>
-      <unit id="org.hamcrest" version="1.1.0.v20090501071000"/>
-      <unit id="org.hamcrest.core" version="1.1.0.v20090501071000"/>
-      <unit id="org.hamcrest.text" version="1.1.0.v20090501071000"/>
```

Comment:
hamcrest is used by tests and should be encouraged to be used so it seems like a bad/weird thing to remove.

removing 1.1 when 1.3 is present I grok - but you are removing more than that hence asking if we know why.

Your links above goes to something about Birt - not sure if that is wrong link ?